hub/examples/image_retraining/retrained_labels.txt

```diff
@@ -7,4 +7,4 @@ uranus
 elliptical
 earth
 venus
-saturn
+saturn
```

Comment:
@Shubby98 Just curious why is it showing a change? Is there whitespace?

Reply:
@ritwik12  yeah there is whitespace after the saturn. actually i dont know how that file modified i never change it when i match it with original file i found order of planet on this file is changed so i just copy paste that order with original file in order.

Reply:
@shubby98 remove whitespace then

Reply:
@ritwik12  i did